### PR TITLE
@oxaudo => Tweak some more analytics for the artwork page

### PR DESCRIPTION
**Pull request by mzikherman**
These are the last follow up issues from @cavvia QA'ing this morning.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit b52050b14ab738b24cf2f74ad0762136bed1adce

https://deploy-preview-1828--artsy-reaction.netlify.com

File: src/Artsy/Analytics/Schema/Values.ts [hunk 1]
```diff
@@ -45,6 +46,8 @@ export enum ActionType {
    */
   ClickedBuyNow = "Clicked buy now",
   ClickedMakeOffer = "Clicked make offer",
+  ClickedContactGallery = 'Clicked "Contact Gallery"',
```
Comment (oxaudo) [inline src/Artsy/Analytics/Schema/Values.ts]: Does this need to be in "" ? seems like the other ones are just plain strings and I think at some point they get converted to the `_` versions. I think it's cleaner not to have double quotes in the names.

File: src/Components/Artwork/Save.tsx [hunk 2]
```diff
@@ -56,15 +62,20 @@ export class SaveButton extends React.Component<SaveProps, SaveState> {
...
-        entity_slug: props.artwork.id,
-      } as Schema.Old)
-  )
+  trackSave = () => {
+    const {
+      tracking,
+      artwork: { is_saved, id, _id },
+    } = this.props
+    const trackingData: SaveTrackingProps = this.props.trackingData || {}
+    const action = is_saved ? "Removed Artwork" : "Saved Artwork"
```
Comment (oxaudo) [inline src/Components/Artwork/Save.tsx]: Is this "Removed Artwork" will be descriptive enough? Is not that essentially "Removed from saved works" ?

File: src/Artsy/Analytics/Schema/Values.ts (hunk 1 above)
Comment (mzikherman) [reply src/Artsy/Analytics/Schema/Values.ts]: I'm just taking it directly from the sheet! I think I got the OK for it.

File: src/Components/Artwork/Save.tsx (hunk 2 above)
Comment (mzikherman) [reply src/Components/Artwork/Save.tsx]: Yea, I'm just keeping what was already here being tracked. The addition was the `context_page` coming thru in `trackingData`.

File: src/Artsy/Analytics/Schema/Values.ts (hunk 1 above)
Comment (oxaudo) [reply src/Artsy/Analytics/Schema/Values.ts]: Then I should change the events above to include the "" as well. Strange to have it both ways one after the other.

Comment (oxaudo) [reply src/Artsy/Analytics/Schema/Values.ts]: But to be a stickler - we don't even capitalize works on the button any more. So it should probably be 'Clicked "Contact gallery"' after all.

**Pull request merged**